MyOwnORM/MyOwnORM/Attributes/CascadeDeleteAttribute.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MyOwnORM
+{
+    [AttributeUsage(AttributeTargets.Property, AllowMultiple = false)]
+    public sealed class CascadeDeleteAttribute : Attribute
+    {
+        public Type TargetType { get; }
```

Comment:
you should not specify here type, because it has already been specified in FK attribute

Reply:
Fixed. Deleted attribute, I don't interact with him in any way.